site/components/src/main/java/uk/nhs/digital/common/feed/RssModifier.java

```diff
@@ -261,11 +256,32 @@ public void modifyEntry(final HstRequestContext context, final Item entry, final
     @NotNull
     private Element getImageElement(String imageUrl) {
         final Element element
-            = new Element("thumbnail", "media", "http://search.yahoo.com/mrss/");
+            = new Element("content", "media", "http://search.yahoo.com/mrss/");
         element.setAttribute("url", imageUrl);
+        addImageProperties(imageUrl, element);
+
         return element;
     }
 
+    private void addImageProperties(String imageUrl, Element element) {
+        String contentType = null;
+        URL url;
+        try {
+            url = new URL(imageUrl);
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+            contentType = connection.getContentType();
+            element.setAttribute("type", contentType.substring(0, contentType.indexOf(";")));
+            BufferedImage bufferedImage = ImageIO.read(url);
+            if (bufferedImage != null) {
+                element.setAttribute("height", String.valueOf(bufferedImage.getHeight()));
+                element.setAttribute("width", String.valueOf(bufferedImage.getWidth()));
+            }
+            element.setAttribute("expression", "full");
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Use log error instead of just printing the stack trace

Reply:
done